src/pdf.rs

```diff
@@ -17,21 +17,21 @@
 
 //! Create PDFs for printed paper backup
 
-use pandoc;
 use std::fs::File;
 use std::io::Write;
 use std::path::{Path, PathBuf};
 
+use itertools::Itertools;
+
 fn read_folder_sorted(folder: &Path) -> Vec<PathBuf> {
-    let mut output_files: Vec<PathBuf> = std::fs::read_dir(folder)
+    std::fs::read_dir(folder)
         .expect("Could not list output directory")
         .map(Result::unwrap)
-        .filter(|file| file.file_name().to_str().unwrap().ends_with("png"))
-        .map(|e| e.path())
-        .collect();
-    // read_dir does not guarantee ordering => explicit sort chunk files
-    output_files.sort();
-    output_files
+        .map(|file| file.path())
+        .filter(|path| path.ends_with("png"))
+        // read_dir does not guarantee ordering => explicit sort chunk files
+        .sorted()
```

Comment:
Awww I thought I looked for this but didn't find it! I wanted the equivalent for python's `sorted(mylist)` to make it inline, but only found `list.sort()` which sucked. Good find, thanks!

Reply:
well, i figured if you're using `itertools`, you might as well use `itertools`...